web-registry/src/components/organisms/BuyCreditsModal/BuyCreditsModal.tsx

```diff
@@ -313,17 +314,35 @@ const BuyCreditsModal: React.FC<BuyCreditsModalProps> = ({
                       type="radio"
                       value="autoretire"
                       checked={values['retirementAction'] === 'autoretire'}
-                      label="Auto-retire credits" // Retire credits now
-                      description="These credits will be retired upon purchase and will not be tradeable. Retirement is permanent and non-reversible."
+                      label="Retire credits now"
+                      description={
+                        <>
+                          {
+                            'These credits will be retired upon purchase and will not be tradeable. Retirement is permanent and non-reversible.'
```

Comment:
```suggestion
                            'These credits will be retired upon purchase and will not be tradable. Retirement is permanent and non-reversible.'
```

For consistency with other PR. although I know this could end up staying the same